clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/MoveExecutorGvcf.scala

```diff
@@ -87,31 +90,35 @@ class MoveExecutorGvcf(moveGvcfCommand: MoveGvcf) extends Executor {
       .map(webClient.ensureOkResponse)
       .flatMap(webClient.unmarshal[Seq[TransferGvcfV1QueryOutput]])
       .map(ensureOnlyOne)
-      .map(_.gvcfPath)
+      .map {
+        _.gvcfPath.getOrElse {
+          throw new Exception(
+            s"The gvcf for Key($prettyKey) has no registered path, and can't be moved."
```

Comment:
Odd spacing and use of parens: `Key($prettyKey)`. I would write `The gvcf for key $prettyKey ...` or `The gvcf for $prettyKey ...` since key is understood from context.

Reply:
Was mirroring the messages elsewhere in the file, but agreed they're weird. I'll sweep through and fix them all up.